source/blog/openshift-openId-integration-with-engine-sso.html.md

```diff
@@ -0,0 +1,126 @@
+---
+title: Configuring Kibana/Elastic Search/Open Shift backend to authenticate with engine sso
+author: rnori
+tags: community, documentation, howto, openshift, ovirt, sso
+date: 2017-04-04 11:40:00 CET
+comments: true
+published: true
+---
+
+Configuring Kibana/Elastic Search/Open Shift backend to authenticate with engine sso.
+
+A step by step instruction on how to configure open shift to authenticate with engine sso.
+
+READMORE
+
+## Background
+
+Kibana/Elastic Search/OpenShift needs to be setup to authenticate using engine so that the users on engine can access Kibana using engine sso to authenticate. 
+
+## Prerequisites
+
+* A fully working and configured oVirt engine instance
+* A fully working and configured instance of Kibana/Elastic Search and Open Shift
+
+## Installing Kibana/Elastic Search and Open Shift backend
+
+Install Kibana/Elastic Search/OpenShift on CentOS7 or RHEL 7.3 as described in http://git.engineering.redhat.com/git/users/rmeggins/test-openshift-heat.git/tree/README-mux.md
```

Comment:
IMO git.engineering.redhat.com is not available outside Red Hat, so we cannot reference it. @richm is there any publicly available howto?